spec/models/refund_spec.rb

```diff
@@ -0,0 +1,199 @@
+require 'rails_helper'
```

Comment:
Thinking about it, Im not sure exporting belongs in a model ?  Isn't that what we use services for in our projects ?

Reply:
Hmm, interesting thought. I'm not sure. I'm calling to_csv which has the same principle as to_h or to_s. Then I'm using presenter, so we don't have that code in the model but in a presenter. But I'm open to suggestions. @leoapost what do you think?

Reply:
I think whether right or wrong (although its never wrong if it works lol) - I guess as long as its consistent with other areas of the system - i.e. if other areas use services then we should use services

Reply:
Ok. let's use service.